docs/source/changelog.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+2021.08.0
+---------
+
+Enhancements
+
+    HTTTP
+
+    - get_file/put_file APIs now support callbacks
+    - New put_file API for transferring data to the remote server (chunked)
+    - Customizable client initializers (through passing ``get_client`` argument)
+    - Support for ``checksum`` (either ``ETag`` or ``MD5-Checksum``) on ``info()`` calls
```

Comment:
ETag maybe needs more explanation?

Reply:
Do we actually check the value anywhere?

Reply:
> Do we actually check the value anywhere?

No, we don't. This is primarily useful for end user consuption (e.g in our use case, we use this to ensure the remote file is not changed).